src/components/table/DataTable.tsx

```diff
@@ -1,157 +1,98 @@
-    name: "Niene Boyen",
-    data: "DEGO DOOK Autobranche",
-    mail: "niene@maasland.nl",
-    verified: true,
-    account: "2 nov 2022",
-    active: "2 nov 2022 11:05",
-    action: (
-      <>
-        <Button writeIcon={true} />
-        <Button lockCloseIcon={true} />
-        <Button binIcon={true} />
-      </>
-    ),
-  },
-  {
-    name: "Ruben Werdmulier Von Elg",
-    data: "DEGO DOOK Autobranche",
-    mail: "ruben@ontwikkelaar.nl",
-    verified: true,
-    account: "2 nov 2022",
-    active: "2 nov 2022 11:05",
-    action: (
-      <>
-        <Button writeIcon={true} />
-        <Button lockCloseIcon={true} />
-        <Button binIcon={true} />
-      </>
-    ),
-  },
-  {
-    name: "Stephan de Preeker",
-    data: "-",
-    mail: "stephan@memory.com",
-    verified: false,
-    account: "2 nov 2022",
-    active: "2 nov 2022 11:05",
-    action: (
-      <>
-        <Button writeIcon={true} />
-        <Button lockCloseIcon={true} />
-        <Button binIcon={true} />
-      </>
-    ),
-  },
-  {
-    name: "Niene Boyen",
-    data: "DEGO DOOK Autobranche",
-    mail: "niene@maasland.nl",
-    verified: true,
-    account: "2 nov 2022",
-    active: "2 nov 2022 11:05",
-    action: (
-      <>
-        <Button writeIcon={true} />
-        <Button lockCloseIcon={true} />
-        <Button binIcon={true} />
-      </>
-    ),
-  },
-  {
-    name: "Ruben Werdmulier Von Elg",
-    data: "DEGO DOOK Autobranche",
-    mail: "ruben@ontwikkelaar.nl",
-    verified: true,
-    account: "2 nov 2022",
-    active: "2 nov 2022 11:05",
-    action: (
-      <>
-        <Button writeIcon={true} />
-        <Button lockCloseIcon={true} />
-        <Button binIcon={true} />
-      </>
-    ),
-  },
-  {
-    name: "Stephan de Preeker",
-    data: "-",
-    mail: "stephan@memory.com",
-    verified: false,
-    account: "2 nov 2022",
-    active: "2 nov 2022 11:05",
-    action: (
-      <>
-        <Button writeIcon={true} />
-        <Button lockCloseIcon={true} />
-        <Button binIcon={true} />
-      </>
-    ),
-  },
-];
+import { MenuItem, InputAdornment, Table, TableRow } from "@mui/material";
+import { Button } from "../button/Button";
 
 export const DataTable = () => {
   const [sortOrder, setSortOrder] = useState("");
   const [sortedTable, setSortedTable] = useState(data);
   const [searchValue, setSearchValue] = useState("");
 
-  const sortOptions = ["Naam", "E-mail", "Geverifieerd"];
+  const sortOptions = [
+    {
+      label: "fullName",
+      value: "Name",
+    },
+    {
+      label: "mail",
+      value: "E-mail",
+    },
+    {
+      label: "verified",
+      value: "Geverifieerd",
+    },
+    {
+      label: "accesTo",
+      value: "Toegang",
+    },
+  ];
+
+  const sortKey = sortOptions.map((entry) => entry.label);
+  const buttonIcons = data.map((entry) => entry.setActions);
+
+  const filteredTable = sortedTable.filter((value) => {
+    if (searchValue === "") {
+      return value;
+    } else if (
```

Comment:
this `else` is not needed if you're returning a value.

```suggestion
    }
```